src/main/flight/pid.c

```diff
@@ -81,20 +81,19 @@ void pidStabilisationState(pidStabilisationState_e pidControllerState)
 
 const angle_index_t rcAliasToAngleIndexMap[] = { AI_ROLL, AI_PITCH };
 
+static filterApplyFnPtr yawNotchFilterApplyFn;
+static void *yawFilterNotch;
 static filterApplyFnPtr dtermNotchFilterApplyFn;
 static void *dtermFilterNotch[2];
 static filterApplyFnPtr dtermLpfApplyFn;
 static void *dtermFilterLpf[2];
-static filterApplyFnPtr ptermYawFilterApplyFn;
-static void *ptermYawFilter;
```

Comment:
I wouldn't have changed the name of these. It's still a `ptermYawFilter` , the name deliberately didn't specify the type of filter, so that the type could be changed without changing this variable name.

The design is such that the code is transparent to the type of filters, and the type can be changed just by changing the `pidInitFilters` function. That's why the filter storage is declared `static` with `pidInitFilters`.

The names of the filters reflect their logical type, not their physical implementation.

So, for example, the DTerm low pass filter could be changed to a FIR-based low pass filter just by changing the code in `pidInitFilters` - no other code needs to change.

As I said, it was designed this way to allow the changes to be kept local to `pidInitFilters`.